#!/usr/bin/env python
PACKAGE = "pcl_tiago"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("setLeafSize_X", double_t, 0, "A double parameter",    0.01, 0,   0.1)
gen.add("setLeafSize_Y", double_t, 0, "A double parameter",    0.01, 0,   0.1)
gen.add("setLeafSize_Z", double_t, 0, "A double parameter",    0.01, 0,   0.1)
gen.add("Max_Iterations", double_t, 0, "A double parameter",    1000, 0,   10000)
gen.add("Distance_Threshold", double_t, 0, "A double parameter",    .01, 0,   20)




exit(gen.generate(PACKAGE, "pcl_tiago", "pcl"))